api/metric/common.go

```diff
@@ -48,12 +48,12 @@ func (m commonMetric) Impl() InstrumentImpl {
 	return m.instrument
 }
 
-func (h commonHandle) recordOne(ctx context.Context, number core.Number) {
-	h.handle.RecordOne(ctx, number)
+func (h commonBoundInstrument) recordOne(ctx context.Context, number core.Number) {
+	h.boundInstrument.RecordOne(ctx, number)
 }
 
-func (h commonHandle) Release() {
-	h.handle.Release()
+func (h commonBoundInstrument) Release() {
```

Comment:
I'll suggest use 'Unbind' to replace 'Release'

Reply:
I was planning to write a brief OTEP on this topic. The spec doesn't mention `Release` at this time. I can be convinced to use `Unbind`.